ntbs-service-unit-tests/DataMigration/NotificationMapperTest.cs

```diff
@@ -253,7 +253,7 @@ private void SetupNotificationsInGroups(params (string, string)[] legacyIdAndLeg
             _migrationRepository.GroupedNotificationsStub = grouped;
         }
         
-        private class MigrationRepositoryStub : IMigrationRepository
+        public class MigrationRepositoryStub : IMigrationRepository
```

Comment:
Why has this become public? Where is it being used that it wasn't before?

Reply:
Ah I was thinking about using this in my tests but went for another approach, will change it back